src/natlinkcore/configure/natlinkconfigfunctions.py

```diff
@@ -167,6 +168,27 @@ def config_remove(self, section, option):
     # def clearUserDirectory(self, arg):
     #     self.clearDirectory('UserDirectory')
         
+    def addPackage(self,packagename):
```

Comment:
Do we want to check if module are installed if adding?
```py
import importlib.util
import sys

# For illustrative purposes.
name = 'itertools'

if name in sys.modules:
    print(f"{name!r} already in sys.modules")
```

Reply:
I don't think so.   pip will just quickly pass over it if it is up to date.  pip does the checking so it is redundant for us to do it again (and pip probably does a better job of determining if something need to happen).

Either way I didn't work on that part of the logic for the PR, just the part tweaking the ini file.

Reply:
Doug, I am not in favour of this change: both Unimacro and Vocola need another directory, in the site-packages, which should be handled by the config program, and not by the loader. 

By setting the "user" directories for Vocola and Unimacro, these are effectively enabled. natlinkconfig and natlinstatus.py handle this nicely enough.

Note: both Unimacro and Vocola need 2 directories for the natlink loader.